src/components/Accordion/Accordion.vue

```diff
@@ -40,12 +40,19 @@ export default {
     title: {
       type: String,
       default: 'Accordion Title'
+    },
+    open: {
+      type: Boolean,
+      default: false
     }
   },
   data () {
     return {
       expanded: false
     };
+  },
+  created () {
+    this.expanded = this.defaultOpen;
```

Comment:
Should this actually be
```
this.expanded = this.open;
```
?

Reply:
Or should the prop be renamed to `defaultOpen`?

Reply:
Probably there should be a test added for this and if there was it would have caught this discrepancy.

Reply:
Oops we changed the prop name right before we pushed, thanks! :sweat_smile: